reference/price_lists.v3.yml

```
@@ -758,11 +758,39 @@ paths:
   '/pricelists/records':
     parameters:
       - $ref: '#/components/parameters/Accept'
-    put: 
+    post: 
       tags:
         - Price Lists Records
       summary: Create Batch of Price Lists Records
-      description: Creates a batch of `Price Lists Records`; may include price list records from more than one price list.  Concurrency limit of 1.
+      description: Creates a batch of `Price Lists Records`; may include price list records from more than one price list. Concurrency limit of 1. Will error out if any record in the batch already exists; use PUT to batch update existing records.
```

```suggestion
      description: Creates a batch of `Price Lists Records`; may include price list records from more than one price list. Concurrency limit of 10, batch size of 100. Will error out if any record in the batch already exists; use PUT to batch update existing records.
```
to match the changes Vivek is working on.